forwards_paper.tex

```diff
@@ -215,31 +215,47 @@ \subsection*{Tree sequences}
 in Figure~\ref{fig:example_tree_sequence}, illustrated by red hexagons. The first
 mutation occurs at site zero (which is in the left-hand tree), which is a simple
 mutation resulting in node $2$ inheriting the state 'T'. The second side (in the
-right hand tree) has two mutations: one occuring over node $3$ changing the state to
+right hand tree) has two mutations: one occurring over node $3$ changing the state to
 `C' and a back mutation over node $1$ changing the state to `G'.
 \jk{Not sure if I'm labouring the point here: this should be reasonably obvious.
 Perhaps if we added the state changes to the trees, and the states of the leaves
 below the samples, we could get rid of all the example discussion here?}
 
-
-\jk{Other stuff we can review here:}
```

Comment:
I'm in favor of reviewing any/all of these things.   Probably can't emphasize this too much.